lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_metadata.rb

```diff
@@ -16,11 +16,11 @@ def self.run(params)
       #####################################################
 
       def self.description
-        'Downloads translated metadata from the translation system'
+        'Downloads translated metadata from the translation system (deprecated)'
       end
 
       def self.details
-        'Downloads translated metadata from the translation system'
+        'Downloads translated metadata from the translation system. This action is deprecated in favor of android_download_translations(…)'
```

Comment:
Also, bummer there's no way to mark an action as deprecated at the Fastlane level 😞

Reply:
I actually didn't double check that … 🤔 did you?

Reply:
Fixed in c725f7a. Thanks for the nudge to look into that!

Note the lines at the end mentioning that the action is deprecated.

```
$ bundle exec fastlane action android_update_metadata
…

Loading documentation for android_update_metadata:

+--------------------------------------------------------------------------------------------------------------------------------+
|                                                    android_update_metadata                                                     |
+--------------------------------------------------------------------------------------------------------------------------------+
| Downloads translated metadata from the translation system                                                                      |
|                                                                                                                                |
| Downloads translated metadata from the translation system. This action is deprecated in favor of android_download_translations |
|                                                                                                                                |
| Created by loremattei                                                                                                          |
+--------------------------------------------------------------------------------------------------------------------------------+

No available options
* = default value is dependent on the user's system

==========================================
This action (android_update_metadata) is deprecated
This action is deprecated in favor of `android_download_translations`
==========================================
```

Reply:
🎉 excellent!